app/components/TimeSeriesChart.tsx

```diff
@@ -110,14 +112,11 @@ type TimeSeriesChartProps = {
   className?: string
   data: ChartDatum[] | undefined
   title: string
-  width: number
-  height: number
```

Comment:
We were passing height and width at every callsite but it had no effect, probably because the `ResponsiveContainer` was overriding it.